addons/io_hubs_addon/components/definitions/morph_audio_feedback.py

```diff
@@ -66,7 +66,8 @@ class MorphAudioFeedback(HubsComponent):
         'node_type': NodeType.NODE,
         'panel_type': [PanelType.OBJECT],
         'icon': 'MOD_SMOOTH',
-        'version': (1, 0, 0)
+        'version': (1, 0, 0),
+        'tooltip': "Use the user's transmitted audio to animate a shape key of the avatar"
```

Comment:
Revised wording that @Heather-Elizabeth-Dodds and I came up with:
```suggestion
        'tooltip': "Use the user's transmitted audio to animate a shape key on the avatar.  Larger numbers cause more movement
"
```

Reply:
isn't 'can optionally' a tautology?